projects/aca-playwright-shared/src/page-objects/components/dataTable/mat-menu.component.ts

```diff
@@ -49,6 +48,13 @@ export class MatMenuComponent extends BaseComponent {
     await menuElement.waitFor({ state: 'detached' });
   }
 
+  async clickMenuItemFromHeaderMenu(menuItem: string): Promise<void> {
+    const menuElement = this.getMenuItemFromHeaderMenu(menuItem);
+    await menuElement.waitFor({ state: 'attached' });
+    await menuElement.click();
```

Comment:
Click action already has auto-waiting for element, so line 53 is not needed.
 
If you want to check which actions have auto-waiting in Playwright, you can read more of it here: https://playwright.dev/docs/actionability

Reply:
fixed